src/components/calcite-action-menu/calcite-action-menu.tsx

```diff
@@ -218,6 +218,7 @@ export class CalciteActionMenu {
 
     return (
       <calcite-popover
+        disablePointer
```

Comment:
Are we ok with this? This will remove the pointer on submenus in the action-bar opening to the right/left. This could be a noticeable difference for users.

Reply:
I think it's good since we're emulating a drop-down.

Reply:
cc @AdelheidF